elasticsearch/README.md

```diff
@@ -0,0 +1,17 @@
+## 0x00
+`sudo -H pip3 install elasticsearch`.
+
+## 0x01
+Download Elasticsearch and Kibana (works with 6.0.0, going to try with newest) from [here](https://www.elastic.co/downloads/past-releases)
+
+## 0x02
+If you downloaded the zip version, unzip `elasticsearch` and move to that `/bin` subdir. Run `./elasticsearch`
+The same for `kibana`.
+You should not see errors, if so: ask us.
+
+## 0x03
+Go to `http://localhost:5200`, `Dev Tools`, copy&paste from `index.json` and click the `green arrow`.
+Go to `Management`, `Index Patterns`, `Create Index Pattern` and digit `tgexport2`. **Don't choose time field**.
+
+## 0x04
```

Comment:
These custom titles seem somewhat unnecessary. You could nest a list like this:
1. Step one.
2. Step two.
 2.1. Substep.
 2.2. Substep.
3. Step three.